R/visualizer-lib-utils.R

```diff
@@ -366,14 +366,12 @@ plot_eval_summary <- function(fit_results, eval_tib = NULL, eval_id = NULL,
       }
     }
     # add theme
-    plt <- plt + pretty_ggplot_theme()
+    plt <- plt + vthemes::theme_vmodern()
```

Comment:
Whether this function ends up in `vthemes` or `simChef`, I think we should make this the optional default via an argument and use base ggplot. Assuming we put `vthemes` in suggests, we can use `rlang::is_installed()` to check for it and ask the user if they want to install if not already available, and if they choose not to then we can fall back to the base theme.

Reply:
This is a good point and ultimately what I want to do in the future. However, do you know if there's a way to set something like a global option that when turned on, always uses `vthemes` or always uses the base theme? I would get really annoyed if I had to write `theme = vthemes` for every visualizer in every simulation.

Reply:
This would be great!